data/pb/sanitation/DIGIT-UI/RoleStatusMapping.json

```diff
@@ -0,0 +1,76 @@
+{
+    "tenantId": "pb",
+    "moduleName": "DIGIT-UI",
```

Comment:
How about this one moduleName @jagankumar-egov are we keeping `DIGIT-UI` or updating into `FSM_UI`